src/Compilers/Test/Core/TestBase.cs

```diff
@@ -235,14 +235,27 @@ public virtual void Dispose()
 
         public static MetadataReference MinAsyncCorlibRef => TestReferences.NetFx.Minimal.minasynccorlib;
 
-        public static MetadataReference ValueTupleRef => TestReferences.NetFx.ValueTuple.tuplelib;
+        public static MetadataReference ValueTupleRef => NetFramework.SystemValueTuple;
 
         public static MetadataReference MsvbRef => NetFramework.MicrosoftVisualBasic;
 
         public static MetadataReference MsvbRef_v4_0_30319_17929 => NetFramework.MicrosoftVisualBasic;
 
         public static MetadataReference CSharpRef => CSharpDesktopRef;
 
+        /// <summary>
+        /// This is a legacy copy of System.ValueTuple. The origin is unclear as this does not appear to be a released
+        /// binary on nuget.org (possible a pre-release copy). This does have a few properties that were interesting
+        /// for a particular style of bug in VS that cannot be reproduced with modern TFMs. Specifically that it 
+        /// dependes on System.Runtime for parts of the impl and can't complie with only a reference to mscorlib. As
+        /// such this is kept around for those tests.
+        ///
+        /// Related issues
+        ///   - https://github.com/dotnet/roslyn/issues/14888
+        ///   - https://github.com/dotnet/roslyn/issues/14267
+        /// </summary>
+        public static MetadataReference ValueTupleLefacyRef => TestReferences.NetFx.ValueTuple.tuplelib;
```

Comment:
>Lefacy [](http://example.com/codeflow?start=50&length=6)

"Legacy"?

Reply:
I'm so proud of this typo that I thought about keeping it.